spinnaker/spinnaker_system/kube_find_image_test.py

```diff
@@ -311,9 +309,9 @@ def make_find_image_stage(self, requisiteStages=[], **kwargs):
 
   def make_deploy_stage(self, imageSource=None, requisiteStages=[], **kwargs):
     bindings = self.bindings
-    cluster = '{app}-{stack}'.format(
+    cluster = str(frigga.Cluster(
```

Comment:
[Here](https://github.com/google/citest/pull/163/files/b350935defc797984c81eee23c36297bb0f561f9..5223587f354f162a4e557d68523d5243df6956b1#diff-a163732ccac0eba8a465d7b7af9c2b4bL338) and [here](https://github.com/google/citest/pull/163/files/b350935defc797984c81eee23c36297bb0f561f9..5223587f354f162a4e557d68523d5243df6956b1#diff-a163732ccac0eba8a465d7b7af9c2b4bL341)